README.rst

```diff
@@ -270,6 +270,9 @@ Choose from the list of available rules:
     the opening brace should be placed on "next" or "same" line after
     classy constructs (non-anonymous classes, interfaces, traits, methods
     and non-lambda functions); defaults to ``'next'``
+  - ``position_after_return_type_hint`` (``'next'``, ``'same'``): whether the opening
```

Comment:
Does `position_after_return_type_hint` option always override `position_after_functions_and_oop_constructs`? Or when arguments list is multiline only?